proofs/rustverifier.go

```diff
@@ -82,8 +79,16 @@ func (rp *RustVerifier) VerifySeal(req VerifySealRequest) (VerifySealResponse, e
 func (rp *RustVerifier) VerifyPoST(req VerifyPoSTRequest) (VerifyPoSTResponse, error) {
 	defer elapsed("VerifyPoST")()
 
-	// flattening the byte slice makes it easier to copy into the C heap
+	// validate verification request
+	if len(req.Proofs) == 0 {
+		return VerifyPoSTResponse{}, errors.New("must provide at least one proof to verify")
+	}
+
+	// CommRs must be provided to C.verify_post in the same order that they were
+	// provided to the C.generate_post
 	commRs := req.SortedCommRs.Values()
+
+	// flattening the byte slice makes it easier to copy into the C heap
```

Comment:
@phritz nit: capitalize and period at the end.

Reply:
@ZenGround0 - I don't see such a rule in any of the go-filecoin markdown docs. Does one exist in the Wiki? 

Note that I always capitalize and add a period if either of these two conditions are met:

1. I'm documenting an exported type
1. I'm documenting an unexported statement with more than one sentence

The line which you have called out meets neither of these two criteria, so I politely decline your request to apply sentencification.

Reply:
NP sounds reasonable, I'll stop bugging people about this.

Reply:
I do appreciate that you evoked the Spirit of Phritz